crypto/fipsmodule/ec/make_tables.go

```diff
@@ -258,6 +264,178 @@ static const alignas(4096) PRECOMP256_ROW ecp_nistz256_precomputed[37] = `
 	return nil
 }
 
+// ------ START ECNISTP TABLES ------
+// Table of multiples of the base point:
+//   [(2j + 1) * 2^(i*(w-1))]G, for i in [0, m - 1] and j in [0, 2^(w-1)-1],
+// where w is window size (hard-coded to 5 for all curves in ec_nistp),
+// and m = ceil(scalar_bit_size / w).
+func generateECNISTPTable(curve elliptic.Curve) [][2]*big.Int {
+	win_size := 5
+	pts_per_subtable := (1 << win_size) >> 1
+	num_subtables := int(math.Ceil(float64(curve.Params().BitSize) / float64(win_size)))
+
+	table := make([][2]*big.Int, 0, num_subtables * pts_per_subtable)
+	for i := 0; i < num_subtables; i += 1 {
+		row := makeOddMultiples(curve, pts_per_subtable, i*win_size)
```

Comment:
`makeOddMultiples` computes odd multiples of `2^shift * G`, where `shift` is the the third argument `i*win_size`. 

But for a fixed `i`, we want `2^(i*w - i)`, no?

Reply:
ah I think the mistake is in the comment above the function, not in the code. We want:
```
//   [(2j + 1) * 2^(i*w)]G, for i in [0, m - 1] and j in [0, 2^(w-1)-1]
```